specifyweb/backend/stored_queries/query_construct.py

```
@@ -32,55 +29,47 @@ def __new__(cls, *args, **kwargs):
 
     def handle_tree_field(self, node, table, tree_rank: TreeRankQuery, next_join_path, current_field_spec: QueryFieldSpec):
         query = self
-        if query.collection is None:
-            raise AssertionError(
-                f"No Collection found in Query for {table}",
-                {"table": table, "localizationKey": "noCollectionInQuery"},
-            )
-        logger.info("handling treefield %s rank: %s field: %s", table, tree_rank.name, next_join_path)
+        if query.collection is None: raise AssertionError( # Not sure it makes sense to query across collections
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)